cassandane/Cassandane/Cyrus/Replication.pm

```diff
@@ -257,6 +257,121 @@ EOF
     }
 }
 
+#
+# Test handling of replication when append fails due to disk error
+#
+sub test_syncall_failinguser
+    :NoStartInstances :min_version_3_5
```

Comment:
I plan to cherry-pick these fixes so it _will_ be on 3.6, otherwise I'd be asking for `:min_version_3_7` here...

```suggestion
    :NoStartInstances :min_version_3_6
```